app/_locales/en/messages.json

```diff
@@ -216,6 +216,9 @@
   "currentConversion": {
     "message": "Current Conversion"
   },
+  "currentLanguage": {
+    "message": "current language"
```

Comment:
Should this start with a capital 'C'?

Reply:
For short phrases and single word translations I've been keeping them lowercased and using CSS to make them uppercase/capitalized, since I thought that would be more flexible. I suppose we should have a discussion on this to figure out what we should do moving forward.